### PR TITLE
sys-cluster/pacemaker: bump to 2.1.2 and EAPI 8

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.29, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		$(use_with snmp)
+	)
+
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	default
+	python_optimize
+	rm -rf "${D}/var/run" "${D}/etc/init.d"
```
Comment (scardracs) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:67]: I don't really like that line but actually it was like that on older version.. If you have a better solution please tell me

Comment (ceamac) [reply sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:67]: What's in `/etc/init.d`, what does it remove?

Comment (ceamac) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:67]:
```suggestion
	rm -r "${ED}/var/run" "${ED}/etc/init.d" || die
```

File: sys-cluster/pacemaker/pacemaker-2.1.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+
+	keepdir /var/lib/pacemaker/{blackbox,cib,cores,pengine}
+	keepdir /var/log/pacemaker/bundles
+
+	find "${D}" -name '*.la' -delete || die
```
Comment (ceamac) [inline sys-cluster/pacemaker/pacemaker-2.1.2.ebuild:74]:
```suggestion
	find "${ED}" -name '*.la' -delete || die
```

**Review by ceamac — CHANGES_REQUESTED** [review]

**Comment by scardracs** [comment]
@ceamac I have no idea on why it is removing /etc/init.d

**Comment by scardracs** [comment]
Looking upstream I've found it's for supports openrc, which isn't supported by default

**Pull request closed without merging**